src/containers/PodcastPage/PodcastListPage.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+// @ts-ignore
+import { OneColumn, Spinner } from '@ndla/ui';
+// @ts-ignore
+import Pager from '@ndla/pager';
+import { RouteComponentProps } from 'react-router';
+import { useLazyQuery } from '@apollo/client';
+// @ts-ignore
+import queryString from 'query-string';
+import { Helmet } from 'react-helmet';
+import { injectT, tType } from '@ndla/i18n';
+import { DefaultErrorMessage } from '../../components/DefaultErrorMessage';
+import { AudioSearch, LocaleType, SearchObject } from '../../interfaces';
+import { podcastSearchQuery } from '../../queries';
+import Podcast from './Podcast';
+
+const DEFAULT_PAGE_SIZE = '5';
+
+export const getPageSize = (searchObject: SearchObject): string => {
+  return searchObject['page-size'] || DEFAULT_PAGE_SIZE;
+};
+export const getPage = (searchObject: SearchObject): string => {
+  return searchObject.page || '1';
+};
+
+interface Props {
+  locale: LocaleType;
+}
+
+const PodcastListPage = ({
+  locale,
+  location,
+  history,
+  t,
+}: Props & tType & RouteComponentProps) => {
+  const [getPodcasts, { error, loading, data }] = useLazyQuery<AudioSearch>(
+    podcastSearchQuery,
+    {
+      fetchPolicy: 'no-cache',
```

Comment:
Hvorfor bruker vi no-cache?

Reply:
Er kanskje ikke noe poeng å aktivt unngå cachet data. Satte `no-cache` mest med tanke på de tilfellene der man laster en side på nytt og cache og database ikke lenger stemmer overens, men dette skjer nok sjeldent og er heller ikke noe stort problem.

Reply:
Ja, tviler på at databasen oppdateres så ofte, men bør heller bruke `cache-and-network` hvis det er en bekymring